send/bbmri_collections

```diff
@@ -36,9 +36,9 @@ cat "$USER_FILE" | timeout -k $TIMEOUT_KILL $TIMEOUT $USER_TRANSPORT_COMMAND
 ERR_CODE=$?
 #Error code 124 means - timed out
 if [ $ERR_CODE -eq 124 ]; then
-	echo "Communication with the peer has timed out with return code: $ERR_CODE (Warning: this error can mask original error 124 from peer!)" >&2
```

Comment:
The warning should stay at the line with a timeout but the description must be better.
The warning on line 52 should be moved to the line 50 in both scripts.